motr/m0crate/crate_index.c

```diff
@@ -553,7 +629,8 @@ static int cr_idx_w_get_nr_keys_per_op(struct cr_idx_w *w, enum cr_opcode op)
 }
 
 static int fill_kv_del(struct cr_idx_w *w,
-		       struct m0_fid *k, struct kv_pair *p, size_t nr)
+		       struct m0_fid *k, struct kv_pair *p, size_t nr,
+				int kpart_one_size, char *kpart_one)
```

Comment:
use consistent formatting between 632 and 633.

Reply:
Fixed.